### ISSUE TITLE
PN-CHORD not working in V2

### ISSUE THREAD

**Issue by DaveBenham** [almostEric/FrozenWasteland#101]
A patch that used ProbablyNote with PN-Chord that worked fine in V1 is not working in V2. I'm getting 4 channels of poly output, but all voltages are the same, except rare cases when one of the voices differs, I presume due to random dissonance kicking in.

**Comment by almostEric** [comment]
Hi, There haven't been any logic changes made to PN-Chord, and it seems like it is working to me:

![image](https://user-images.githubusercontent.com/457423/144623522-8834cc52-
7540-4df1-8000-3314e30660dd.png)

Maybe you can share your patch with me

**Comment by DaveBenham** [comment]
I figured out how to reproduce the issue.
If I open a V1 patch that contains PN, then all notes are red for all scales. PN-CHORD does not work in that circumstance.
I can manually set notes green, and then PN and PN-CHORD work properly, but only for the current scale. The other scale's notes are still red.

The patch is easily resurrected by deleting the PN and replacing it with a new instance while running V2.

I posted a topic at https://community.vcvrack.com/t/probably-note-scale-bug-opening-v1-patch-in-v2/15139?u=davevenom that contains an example patch and image.
